reference/track-group.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: "Track Group"
+api: "POST https://api.promptlayer.com/rest/track-group"
+---
+
+Associate a group with a request.
+
+## Example Code
+```python
+pl_group_id - promptlayer.group.create()
+
+import requests
+response = requests.post(
+  "https://api.promptlayer.com/rest/track-group",
+  json={
+      "api_key": "<YOUR_API_KEY>",
+      "request_id": "<REQUEST_ID>",
+      "group_id": pl_group_id,
```

Comment:
Why is group ID called `pl_group_id` but request ID is `<REQUEST_ID>`? Please keep consistent.